src/scenic/domains/driving/roads.py

```diff
@@ -645,6 +647,8 @@ class Intersection(NetworkElement):
     outgoingLanes: Tuple[Lane]
     maneuvers: Tuple[Maneuver]  # all possible maneuvers through the intersection
 
+    signals: Tuple[Signal]
```

Comment:
Are these in any particular order? My plan had been for these to align with `Intersection.incomingLanes` (which the corresponding entry for a lane being `None` if there is no signal for it), but I can also see an argument for simply having a list of all signals. If we go with the latter, then we should also allow getting the signal(s) corresponding to a given incoming lane and the signal corresponding to a given maneuver.

Reply:
No. Right now they aren't in any particular order. I agree that having them align with `Intersection.incomingLanes` could be very useful. I think it could be reasonable to have both approached implemented. As I've said above, we could introduce this features in a future PR.